beehive.go

```diff
@@ -106,7 +105,7 @@ func main() {
 		log.Fatalf("Error creating the configuration %s", err)
 	}
 
-	if config.URL().String() != cfg.DefaultPath() { // the user specified a custom config path or URI
```

Comment:
String() does some encoding when using Windows paths which is not what we want here.